new-dti-website/components/apply/Banner.tsx

```diff
@@ -14,6 +14,23 @@ const variantStyles: Record<NonNullable<BannerProps['variant']>, string> = {
   success: 'bg-black border-b-2 border-t-2 border-t-neutral-800 border-b-neutral-800'
 };
 
+/**
+ * `Banner` Component - Displays a customizable banner at the top of the page.
+ *
+ * @remarks
+ * The `Banner` component shows a message in a banner that can be styled and positioned
+ * based on the user's scroll position. It supports three variants (`info`, `alert`, and `success`), which can be modified
+ * (I just used these variant names because they're the most common banner usages),
+ * and allows additional customizations via className.
+ *
+ * @param props - Contains:
+ *   - `message` (string): The message to be displayed in the banner. *(Required)*
+ *   - `variant` (`'info' | 'alert' | 'success'`, optional): Specifies the banner's style. Defaults to `'info'`.
```

Comment:
i would rename the variants because i think they're a bit vague and don't exactly correspond to the actual visual style of the banner.

I'd do this:

|Image|Current|Proposed|
|-|-|-|
|<img width="1102" alt="Screenshot 2024-11-26 at 10 09 22 AM" src="https://github.com/user-attachments/assets/28a2e223-00be-425d-8476-66a041fac437">|`alert`|`accent`|
|<img width="1102" alt="Screenshot 2024-11-26 at 10 08 38 AM" src="https://github.com/user-attachments/assets/3789235f-a37a-44ad-9c6b-6f0636905ff8">|`info`|`default`|
|<img width="1102" alt="Screenshot 2024-11-26 at 10 08 58 AM" src="https://github.com/user-attachments/assets/3ec424f5-9835-437a-a120-f5aa8028afca">|`success`|`outlined`|